tests/e2e/config/jest.setup.js

```diff
@@ -35,6 +35,15 @@ async function trashExistingPosts() {
 	}
 }
 
+/**
```

Comment:
I just realized this needs to be moved to core tests or utils package then called in the search test itself. This PR works great for us but package users wouldn't have `WP_VERSION` in their environment. Sorry I missed that the first time.

Reply:
No worries at all! I'll go ahead and make those changes :+1:

Reply:
I updated the function to return an object with the WP and WC versions instead of update the `process.env`. My thinking is this could be more useful in other tests, if needed, too. Let me know what you think!